src/RefreshButton.scss

```diff
@@ -0,0 +1,56 @@
+@import 'variables';
+
+.RefreshButton-svg {
+    fill: #000;
+    width: 1.7rem;
+    height: 1.7rem;
+}
+
+button.btn.RefreshButton--disabled {
+    background-color: $c2lc-primary-button-color;
+    cursor: default;
+}
+
+button.btn.RefreshButton svg {
+    stroke: #000;
+}
+
+button.btn.RefreshButton--disabled svg {
+    stroke: $c2lc-disabled-color;
+}
+
+button.btn.RefreshButton {
+    margin-top: 0.5rem;
+    padding: 0.6rem 0.6rem 0.6rem 0.6rem;
+    border: 1px solid #FFF;
+    border-radius: 50%;
+    background-color: #FFF;
+}
+
+button.btn.RefreshButton:not(:focus) {
+    box-shadow: 0 0 8px rgba(0, 0, 0, 0.25);
+}
+
+button.btn.RefreshButton.RefreshButton--disabled:active:focus {
+    background-color: $c2lc-primary-button-color;
+    box-shadow: 0 0 0 $c2lc-focus-indicator-width $c2lc-focus-indicator-color;
+}
+
+button.btn.RefreshButton--pressed {
+    background-color: #000;
+}
+
+button.btn.RefreshButton.RefreshButton--pressed.RefreshButton--disabled,
+button.btn.RefreshButton.RefreshButton--pressed.RefreshButton--disabled:active:focus {
+    background-color: $c2lc-disabled-color;
+}
+
+button.btn.RefreshButton--pressed svg,
+button.btn.RefreshButton--pressed.RefreshButton--disabled svg {
+    stroke: #FFF;
+}
+
+button.btn.RefreshButton.RefreshButton__run-button--disabled:active:focus {
```

Comment:
I think this rule isn't needed. I couldn't see where we were using the `RefreshButton__run-button--disabled` class.